asyncpg/connection.py

```diff
@@ -1792,7 +1793,8 @@ async def connect(dsn=None, *,
                   direct_tls=False,
                   connection_class=Connection,
                   record_class=protocol.Record,
-                  server_settings=None):
+                  server_settings=None,
+                  target_session_attribute=SessionAttribute.any):
```

Comment:
For improved libpq compatibility let's add support for the `PGTARGETSESSIONATTRS` environment variable also.  Hence, the default here should be `None` (unspecified).